larwirecell/Components/CookedFrameSource.h

```diff
@@ -42,6 +43,16 @@ namespace wcls {
     double m_tick;
     int m_nticks;
     std::vector<std::string> m_frame_tags;
+    art::InputTag m_wiener_inputTag;
+    art::InputTag m_gauss_inputTag;
+    std::vector<art::InputTag> m_badmasks_inputTag;
+    art::InputTag m_threshold_inputTag;
+    bool m_cmm_setup{false}; // true means use cmm setup
+    std::string m_wiener_tag{"wiener"};
+    std::string m_gauss_tag{"gauss"};
```

Comment:
This is all too specific and must be made more general.

The user may have all different manner of frame and trace tags and there should not be a modal `cmm_setup` flag.  The "cooking" should all be data-driven.

To make tracking these changes easier I suggest closing this PR (no merge) and updating #36.

I'll add more comments in PR #36.